src/db/seed.py

```
@@ -1,196 +1,303 @@
+                losses=0,
+                aliases=[t["name"].split()[-1], t["abbrev"]],
+                season_stats=t["stats"]
             )
             session.add(team)
     session.flush()
     print("NBA teams seeded")
 
 
-def seed_nfl_players(session, nfl_league):
-    """Seed sample NFL players with stats."""
+def seed_nfl_players(session, league_id: str):
```

Ruff (0.14.14)

Unused function argument: `league_id`

(ARG001)